src/component/Icon/ChevronRight.tsx

```
@@ -0,0 +1,23 @@
+import type { IconProps } from "./types";
+
+const ChevronRight = ({ size = 24, className }: IconProps) => {
+  return (
+    <svg
+      width={size}
+      height={size}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)